app/helpers/progress_tab_helper.rb

```diff
@@ -29,4 +29,13 @@ def create_badge_array(current_value)
 
     badges.reverse
   end
+
+  def numbers_to_relative_percentages(array)
+    max_value = array.max
+    array.map { |value| ((value.to_f / max_value.to_f) * 100.0).to_i }
+  end
+
+  def set_min_value(value, min_value)
+    value < min_value ? min_value : value
```

Comment:
[Style] 
```suggestion
    [value, min_value].max
```

Got confused here a bit, since the function returns the greater of the two values, but is called `set_min_value`

Reply:
Agreed! Much easier to read (and no need to create a helper function). Thanks Govind!